docs/how-to-guides/develop-a-prompty/index.md

```diff
@@ -0,0 +1,384 @@
+# Prompty
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+Promptflow provides the `prompty` feature to provide customers with a simple way to develop prompt template.
+
+## Create a prompty
+In promptflow, a file with a `.prompty` extension will be treated as a prompty.
+
+### Prompty specification
+The prompty asset is a markdown file with a modified front matter. 
+The front matter is in yaml format that contains a number of metadata fields which defines model configuration and expected inputs of the prompty.
```

Comment:
explain the content is a jinja tempalate?

Reply:
add explain that jinja format